src/views/Dashboard.vue

```diff
@@ -263,24 +263,24 @@ export default {
     },
 
     /**
-     * Searches the filings history for a 'paid' status (ie, not yet completed).
+     * Searches the filings history for a "pending" state, ie, paid (not yet completed).
      * Used to block new filings while there's a pending one.
-     * @param filings The array of filings in history
+     * @param filings the array of history filings array to search
      */
-    checkPendingFilings (filings: Array<any>): boolean {
-      if (!filings || !filings.length) return // safety check
-
-      const foundCoa = filings.some(filing => {
-        if (filing.name === FilingNames.ADDRESS_CHANGE && filing.isPaid) {
-          this.coaPending = true
-          this.coaEffectiveDate = filing.effectiveDate
+    checkPendingFilings (filings: Array<any>): void {
+      if (!filings?.length) return // safety check
+
+      const foundPaid = filings.some(filing => {
+        if (filing.isPaid) {
+          if (filing.isBcompCoaFutureEffective) {
+            this.coaPending = true
+            this.coaEffectiveDate = filing.effectiveDate
+          }
           this.hasBlockerFiling = true
```

Comment:
Now, if any filing is status Paid, we consider that a blocker filing. (Previously checked for only BCOMP and only COA.)